### ISSUE TITLE
[1.19.2] EU doesn't transfer between machines

### ISSUE THREAD

**Issue by Ingvin** [TinyModularThings/IC2Classic#607]
For first you are awesome, I used to play with the old IC2 with 1.4.x-1.5.x versions and I am really happy that you ported it to 1.19.

Back to the problem: few days ago I decided to setup my own private server and I was looking for IC2 alternatives and found IC2 classic. I installed clean official java client (1.19.2-release) via microsoft's launcher, installed Forge 43.2.4 via their jar installer and copied the last version of mod (2.0.3) into the mods directory.

The game starts up with no errors in the logs and I tried to check if everything works, so I created new world in creative and build a generator, two copper cables and a battery block but the battery block was not charging. Then I tried to charge any battery item in the generator and it worked.

So I tried few more times with this build and different versions of Forge and the mod but it still doesn't work. Today I tried the same with Windows (before that I tested everything on a Linux system) and got the same problem.

Can you help me with this? Maybe I'm doing something wrong?

**Comment by Meduris** [comment]
could you provide screenshots of the setup you tried? the batbox, mfe etc all have one side that is marked like this which is only for energy output

![2023-02-02_14 32 14](https://user-images.githubusercontent.com/15174539/216338643-0224ff07-cc79-4641-a32a-578c93bcdbdf.png)

**Comment by Ingvin** [comment]
Oh thanks, I looked at your screen and realized that I'm dumb and I need to sleep. 😅 I did a search in the creative window with blocks and typed "bat" in the search bar and I used the wrong block. Instead of BatBox I used Battery Station and thought it just has a new texture and was too lazy to try other machines. Now I looked at the screenshot, carefully read the names of the blocks and opened the wiki about Battery Stations...

Really ashamed about that, thank you again :)
